### ISSUE TITLE
Unable to have slide content fill available slide height

### ISSUE THREAD

**Issue by p45mark** [FormidableLabs/nuka-carousel#684]
# Bugs and Questions

### Describe Your Environment

- What version of nuka-carousel are you using?
4.6.6
- What version of React are you using?
16.13.1
- What browser are you using?
Chrome
- What machine are you on?
Mac Mojave
### Describe the Problem

https://codesandbox.io/s/brave-khorana-bowrz?file=/src/index.js

**Expected behavior:** 
I expect setting the div for my slide to height = 100% would allow my content to fill in all available space 
 
**Actual behavior:** [What actually happens]
The height of the slide expands depending on it's content to more than double the height of the slide with the most content 

### Additional Information
This only seems to show up when refreshing the page, as opposed to changing the content height to 100% and the page refreshing via HMR. It appears to be a problem with how height is calculated but I could use some help tracking it down.

If this isn't a bug with how the slide height is calculated, I'd still appreciate any suggestions as to how I can get the content of each slide to stretch to fit the slide. I have a post on [StackOverflow too](https://stackoverflow.com/questions/61128521/unable-to-set-nuka-carousel-slide-content-height)
 

```
const Item = ({ children }) => (
  <div
    style={{
      boxShadow: "rgb(153, 153, 153) 2px 2px 4px 2px"
      // height: "100%"
      // height: "100vh"
      /// height: 'inherit'
    }}
  >
    {children}
  </div>
);

<Carousel
    wrapAround
    heightMode="max"
    slidesToShow={1.25}
    cellAlign="center">
        {/* Item components with varying content */}
</Carousel>
```
### Without any height value set:
![image](https://user-images.githubusercontent.com/48729903/78930848-a361e000-7a59-11ea-89f7-1877eea6dbf5.png)

### With height set to 100%
![image](https://user-images.githubusercontent.com/48729903/78931094-2daa4400-7a5a-11ea-9010-6186fe673763.png)

**Comment by sarmeyer** [comment]
Hi @p45mark - Do you mean to use `heightMode={current}`? 

```
Change the height of the slides based either on the first slide, the current slide, or the maximum height of all slides. Overrides height set by initialSlideHeight
```

**Comment by sarmeyer** [comment]
@p45mark Did the above comment help you solve your issue?

**Comment by theredwillow** [comment]
`heightMode={'current'}` doesn't do anything at all for me.

I am using gatsby-images inside of `<a>` external link tags as my slides. Some slides are too big, others are too small and aligned to the top.

Code [here](https://github.com/theredwillow/chili-when-its-chilly/blob/master/src/components/Sponsors/index.jsx), live demo [here](https://chili-when-its-chilly.netlify.app/) (see Sponsors section at bottom).

**Comment by ValGeorgiev** [comment]
Hey @p45mark thanks for raising this with as, this should be fixed with the latest release 5.0.14. I will close this for now, please let me know if you have any other issues.
